<?xml version="1.0"?>
<launch>
  <!--Database for poses, actions, etc-->
  <rosparam param="database_location">"~/db/flo.db"</rosparam>
  <node name="pose_db_manager" pkg="flo_core" type="flo_db.py" respawn="true"/>
  <!--Game Runer-->
  <node name="game_runner" pkg="flo_core" type="game_runner.py" respawn="true"/>
  <!--Logging-->
  <include file="$(find flo_core)/launch/recording.launch"/>
  <!--System Monitoring-->
  <node name="system_monitor" pkg="system_monitor" type="stats_publisher.py" respawn="true"/>
</launch>
